Okta.AspNetCore.Test/OpenIdConnectOptionsHelperShould.cs

```diff
@@ -56,6 +59,27 @@ public void SetOpenIdConnectsOptionsCorrectly(bool getClaimsFromUserInfoEndpoint
             oidcOptions.GetClaimsFromUserInfoEndpoint.Should().Be(oktaMvcOptions.GetClaimsFromUserInfoEndpoint);
             oidcOptions.CallbackPath.Value.Should().Be(oktaMvcOptions.CallbackPath);
 
+            var jsonClaims = oidcOptions
+                .ClaimActions.Where(ca => ca is JsonKeyClaimAction)
+                .Cast<JsonKeyClaimAction>();
+
+            if (useIdentityClaims)
```

Comment:
This if/else block feels awkward in a unit test.  Can the assertions be broken out in a more intuitive way?  Seems the arguments to the test method itself potentially effect the behavior of the test; I would consider the use of the inlinedata attributes for data that is operated on, not for logical forking.

Reply:
I agree with you, this did not looked too good to me as well. The thing is we have the function to test OpenIDConnect options and the new option belongs here but this testing looks cumbersome. I will create another test function just for this new option.